pkgs/development/python-modules/icontract/default.nix

```diff
@@ -60,18 +59,24 @@ buildPythonPackage rec {
   ];
 
   disabledTestPaths = [
-    # mypy decorator checks don't pass. For some reaseon mypy
+    # mypy decorator checks don't pass. For some reason mypy
     # doesn't check the python file provided in the test.
     "tests/test_mypy_decorators.py"
   ];
 
+  # Upstream adds some plain text files direct to the package's root directory
+  # https://github.com/Parquery/icontract/blob/master/setup.py#L63
+  postInstall = ''
+    rm -f $out/{LICENSE.txt,README.rst,requirements.txt}
+  '';
```

Comment:
The reason I am removing those files is because since they're in the root of the `$out` directory, if you put two packages that expose the same file (e.g.: `README.rst`), like `pylddwrap` and `icontract`, this will cause collisions in Python's `buildEnv`.

Reply:
Thanks for the explaination :+1: